src/mol-plugin-state/transforms/model.ts

```diff
@@ -375,6 +400,49 @@ const TrajectoryFromXYZ = PluginStateTransform.BuiltIn({
     }
 });
 
+type TrajectoryFromLammpsData = typeof TrajectoryFromLammpsData
+const TrajectoryFromLammpsData = PluginStateTransform.BuiltIn({
+    name: 'trajectory-from-lammps-data',
+    display: { name: 'Parse Lammps Data', description: 'Parse Lammps Data string and create trajectory.' },
+    from: [SO.Data.String],
+    to: SO.Molecule.Trajectory,
+    params: {
```

Comment:
This should be PD.Select with the options from https://docs.lammps.org/units.html. Maybe call it unitsStyle in line with the lammps docs?